v19.1/online-schema-changes.md

```diff
@@ -13,6 +13,10 @@ Benefits of online schema changes include:
 - Your application's queries can run normally, with no effect on read/write latency. The schema is cached for performance.
 - Your data is kept in a safe, [consistent][consistent] state throughout the entire schema change process.
 
+{{site.data.alerts.callout_danger}}
+Schema changes consume additional resources, and if they are run near the cluster's peak capacity, latency spikes can occur. This is especially true for any schema change that adds columns, drops columns, or adds an index. We do not recommend doing more than one schema change at a time while in production.
```

Comment:
> if they are run near the cluster's peak capacity

Suggest "if they are run when the cluster is near peak capacity"

Otherwise LGTM